staging/src/k8s.io/controller-manager/OWNERS

```diff
@@ -0,0 +1,23 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- andrewsykim
+- cheftako
+- deads2k
+- lavalamp
+- liggitt
+- mtaufen
+- sttts
+reviewers:
+- andrewsykim
+- cheftako
+- deads2k
+- lavalamp
+- liggitt
+- luxas
+- mtaufen
+- sttts
+- sig-cloud-provider-leads
```

Comment:
`sig-cloud-provider-leads` needs to be added at https://github.com/kubernetes/kubernetes/blob/master/OWNERS_ALIASES

Reply:
Fixing this seems better done in a separate PR. Removing reference for now.